### DISCUSSION TITLE
İlk 1000 için kaç puan gerekiyot

### DISCUSSION THREAD

**Discussion by Sampiyonbjk** [IBAX-io/go-ibax#1539] [Soru Cevap - Türkçe]
İlk 1000  için kaç puan gerekiyor

**Comment by Sampiyonbjk** [comment]
26.000 sıralamaya girermi

**Comment by Yusufknk** [comment]
Bunun için bir puan sınırı yok. Bunu ilk bindekilerin puanı belirler.

**Reply by Sampiyonbjk** [reply]
Son sandık umarım iyi çıkar
